src/java.base/share/classes/java/util/Calendar.java

```diff
@@ -3417,7 +3417,7 @@ private void updateTime() {
 
     private int compareTo(long t) {
         long thisTime = getMillisOf(this);
-        return (thisTime > t) ? 1 : (thisTime == t) ? 0 : -1;
+        return Long.compare(thisTime, t);
```

Comment:
Probably, in this case `thisTime` variable can also be dropped.

Reply:
Inlined